scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -2156,15 +2143,22 @@
           (util/lazy-load-var fn-name-from-config)
           job->acceptable-compute-clusters)
         pool->resources-atom (pool-map
-                               pools'
-                               (fn [{:keys [pool/name]}]
-                                 (make-offer-handler
-                                   conn (get pool-name->fenzo-state name)
-                                   pool-name->pending-jobs-atom agent-attributes-cache
-                                   fenzo-max-jobs-considered fenzo-scaleback fenzo-floor-iterations-before-warn
-                                   fenzo-floor-iterations-before-reset (get pool->match-trigger-chan name)
-                                   rebalancer-reservation-atom mesos-run-as-user name
-                                   cluster-name->compute-cluster-atom job->acceptable-compute-clusters-fn)))]
+                              pools'
+                              (fn [{:keys [pool/name]}]
+                                (make-pool-handler
```

Comment:
Would have liked to not have this reformat. E.g., in 'git gui' don't include the lines that are just reformats. Now its MUCH harder to determine what variables changed. It *looks* like nothing changed. True? 

Suggestion to turn off automatic reformatting.

Reply:
I actually reformatted this myself because it is, personally, so hard to visually separate the 13 parameters to this function. But uh, I get it, the diff would have nicely highlighted the changes in line. I will change back but again, so hard to read this.